packages/veritone-react-common/src/components/Video/RestartMediaButton/index.js

```diff
@@ -0,0 +1,29 @@
+import { string, func, shape } from 'prop-types';
+import React, { Component } from 'react';
+import classNames from 'classnames';
+
+export default class RestartMediaButton extends Component {
```

Comment:
I'm wondering if this really needs to be a component. It seems pretty specific to the Video component and is only used in one place.